app/controllers/candidate_interface/unsubmitted_application_form_controller.rb

```diff
@@ -9,6 +9,7 @@ def before_you_start; end
     def show
       @application_form_presenter = CandidateInterface::ApplicationFormPresenter.new(current_application)
       @application_form = current_application
```

Comment:
Possibly OT: does it seem odd to hang on to the presenter-wrapped and the original, unwrapped object? I would expect to see just the wrapped one.

Reply:
Yes, very. This was the thing I thought might be (at least partially) contributing to the high object allocation flagged by Skylight.

Reply:
great